content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -236,19 +187,19 @@ Run the following commands on the virtual machine you want to add to the Istio m
 1. Install `cluster.env` within `/var/lib/istio/envoy/`.
 
     {{< text bash >}}
-    $ sudo cp ~/cluster.env /var/lib/istio/envoy/cluster.env
+    $ sudo cp "${HOME}"/cluster.env /var/lib/istio/envoy/cluster.env
     {{< /text >}}
 
 1. Add the istiod host to `/etc/hosts`.
 
     {{< text bash >}}
-    $ sudo cp ~/hosts /etc/hosts
+    $ sudo cp "${HOME}"/hosts /etc/hosts
```

Comment:
this does not add istiod to /etc/hosts, it replaces /etc/hosts. This is not a desireable model. Please use the addendum model as per prior documentation.